_config.yml

```diff
@@ -1,95 +1,47 @@
-# Website title - used in the <title> tag of the <head> section of the website.
 title: Linaro Connect
-# Name of the site - used to identify the site for setting the active class in the universal nav
-name: Linaro Connect
-# GitHub branch for netlify CMS config.yml file.
-branch: develop
-# Description of the website used in the Meta Description.
-description: > # this means to ignore newlines until "baseurl:"
+url: https://connect.linaro.org
+baseurl: ""
+description: |-
   Hundreds of the world’s best Linux on Arm developers come to Linaro Connect each time because they know it is the leading place to meet with the global community and to learn about what is going on in the industry.
-# Destination of the Jekyll site upon build of site.
 destination: _site
-# Code of current connect.
-connect-code: SAN19
-connect-text: Linaro Connect San Diego 2019
-previous-connect-code: BKK19
-email: contact@linaro.org
-# Status of current Connect
-connect-status: post
-# Include the Linaro Jekyll Theme
-theme: jumbo-jekyll-theme
-# Permalink for the blog posts of the Jekyll website.
 permalink: /resources/:categories/:slug/
-# Syntax highlighter for 96boards blog
+theme: jumbo-jekyll-theme
 highlighter: rouge
-# Markdown version
 markdown: kramdown
-blog_images_dir: assets/images/content/
-# production status for robots.txt and files alike.
-production: false
 kramdown:
   input: GFM
-  syntax_highlighter: rouge # Syntax highlighter for Kramdown.
-# Encoding of the files being included in the Jekyll Build.
+  syntax_highlighter: rouge
 encoding: utf-8
-# Jekyll configuration for running the site in "Safe" mode.
 safe: false
-# Search label setting for site search with e.g. search/?fields.label=96Boards
-search-label: Linaro
-# Disqus Comments settings.
-disqus: true
-disqus_shortname: 'linaroconnect'
-# This is the url that is used when your site is built for canonicals etc.
-url: "https://connect.linaro.org"
-# Make sure that Liquid errors stop the build
-liquid:
-    error_mode: strict
-# Shema.org Setup -> used in the schema.html include.
-schema:
-    organisation:
-        name: Linaro Connect
-        url: https://www.connect.linaro.org
-        sameAs:
-            - https://twitter.com/linaroorg
-            - https://www.facebook.com/LinaroOrg
-            - https://www.linkedin.com/company/1026961/
-            - https://plus.google.com/+LinaroOnAir
-            - https://github.com/Linaro
-            - https://www.slideshare.net/linaroorg
-        address:
-            street: Harston Mill, Royston Rd
-            town: Harston
-            county: Cambridge
-            post_code: CB22 7GG
-            country_code: UK
-# Files that should be excluded from the Jekyll build.
 exclude:
-  - _site
-  - staging.connect.linaro.org
-  - production.connect.linaro.org
```

Comment:
You've done it again and removed important exclusion lines.

Reply:
Ah - staging and production are lower down the list now. But `_site` should be kept unless we agreed to remove it from linaro.org as well?